instrumentation/instacosmos/insta_cosmos.go

```diff
@@ -0,0 +1,80 @@
+// (c) Copyright IBM Corp. 2024
```

Comment:
More like a suggestion. How about rename this file to `instacosmos.go` for consistency :)

Reply:
done 👍